test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3109,6 +3109,42 @@ def setup_rnn_tests():
 
 setup_rnn_tests()
 
+# Unary floating ufunc tests
+# Tests that integral arguments are properly cast to float
+unary_floating_ufuncs = (
+    'sin', 'cos', 'tan',
+    'asin', 'acos', 'atan',
+    'sinh', 'cosh', 'tanh',
+    'ceil', 'floor',
+    'exp', 'expm1', 'log', 'log10', 'log1p', 'log2',
+    'sqrt', 'rsqrt',
+    'trunc',
+    'erf', 'erfc', 'erfinv',
+    'lgamma', 'digamma',
+    'sigmoid'
+)
+
+def setup_unary_floating_ufunc_tests_helper(cls, op_name):
+    op = getattr(torch, op_name)
+
+    def test_fn(self):
+        class module(torch.nn.Module):
+            def forward(self, x):
+                return op(x)
+
+        x = (torch.rand(2, 3, 4) * 10).to(torch.int)
```

Comment:
Let's switch torch.int to torch.long. That was my mistake.

Reply:
Sure! Should be resolved here: https://github.com/pytorch/pytorch/pull/33322/commits/49ec5a40809f4f496b2850ab6a45579f92c8f520.